crates/bevy_window/src/window.rs

```diff
@@ -514,31 +595,31 @@ impl WindowResolution {
         self
     }
 
-    /// The window's client area width in logical pixels.
+    /// The window's area width in logical pixels.
```

Comment:
Actually I think "client area" was the proper term, at least on Windows.

Reply:
Reverted that part. I deleted it in the `WindowResolution`'s methods but not in the `Window`'s ones anyway...